_posts/0000-01-03-dumpsterfire.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Happy Thanksgiving! 
 
-![droidtocat](https://octodex.github.com/images/droidtocat.png)
+![gobble-o-tron](https://octodex.github.com/images/gobbleotron.gif)
```

Comment:
This was a great change, very inline with the holidays 👍

Reply:
Thanks @beardofedu , your feedback is always so valuable!